modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/lineartransform/v0.1.0.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/lineartransform/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "import": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0",
+    "{{base}}/schemas/neurosciencegraph/atlas/transform/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:LinearTransformShape",
+      "@type": "sh:NodeShape",
+      "description": "Linear transform including rigid and affine transform, described by a 3x4 matrix",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:LinearTransform",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/atlas/transform/v1.0.0/shapes/TransformShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:linearTransformType",
```

Comment:
I suggest to target two classes in the shape this:LinearTransformShape:
"targetClass": ["nsg:LinearTransform","nsg:RigidLinearTransform","nsg:AffineLinearTransform"]

so that this property shape (with "path": "nsg:linearTransformType") can be removed.